controller/src/mantarray_desktop_app/main_process/server.py

```diff
@@ -1104,12 +1100,13 @@ def after_request(response: Response) -> Response:
                 response_json["root_recording_path"]
             )
     msg = "Response to HTTP Request in next log entry: "
-    if response.status_code == 200:
+    if response.status_code == 200 and "system_status" not in rule.rule:
         # Tanner (1/19/21): using json.dumps instead of an f-string here allows us to perform better testing of our log messages by loading the json string to a python dict
         msg += json.dumps(response_json)
-    else:
+        logger.info(msg)
+    elif response.status_code != 200:
         msg += response.status
-    logger.info(msg)
+        logger.info(msg)
```

Comment:
We now how 2 lines identical lines: `logger.info(msg)`. It would be better to just have it once and guard it with and `if` if necessary